homeassistant/components/switch/light.py

```diff
@@ -16,7 +16,7 @@
 from homeassistant.helpers.event import async_track_state_change
 from homeassistant.helpers.typing import ConfigType, HomeAssistantType
 
-from . import PLATFORM_SCHEMA, Light
+from homeassistant.components.light import PLATFORM_SCHEMA, Light
```

Comment:
We're moving this platform from one entity component to another. ☺

Reply:
Yes, but this is the "correct" one. It's a bit weird, but it actually feels right. The switch integration knows how to distinguish itself on being a light.